docs/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "wikit-docs",
```

Not that it matters a whole lot, but maybe for consistency's sake:
```suggestion
  "name": "@wmde/wikit-docs",
```